<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmDeviceSelection</class>
 <widget class="QMainWindow" name="FrmDeviceSelection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>410</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>540</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_Practice">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>681</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Công cụ thực hành</string>
    </property>
    <widget class="QListWidget" name="listWidget_toolPractice">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>661</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_cancel">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>375</y>
      <width>100</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Thoát</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_Display">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>681</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Công cụ hiển thị dữ liệu</string>
    </property>
    <widget class="QListWidget" name="listWidget_toolDisplay">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>661</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_Name">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>701</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(25, 149, 173);
color: rgb(255, 255,255);</string>
    </property>
    <property name="text">
     <string>Cảm biến</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_Control">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>681</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Công cụ điều khiển</string>
    </property>
    <widget class="QListWidget" name="listWidget_toolControl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>661</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>listWidget_toolDisplay</tabstop>
  <tabstop>listWidget_toolPractice</tabstop>
  <tabstop>pushButton_cancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
